debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -728,6 +731,21 @@ private static void dumpClassJ9ExtraModifiers(PrintStream out, long accessFlags)
 			out.append("(sealed) ");
 	}
 
+	private static void dumpMethodJ9Modifiers(PrintStream out, long modifiers, long extraModifiers) {
+		if ((modifiers & J9JavaAccessFlags.J9AccMethodCallerSensitive) != 0) {
+			out.append("(@CallerSensitive) ");
```

Comment:
nit: Please remove the trailing space from the call site, and make the space a prefix (instead of a suffix) in the messages here.